src/lib.rs

```diff
@@ -16,7 +16,7 @@ use std::str::from_utf8;
 
 use crate::error::{clear_error, handle_c_error, set_error};
 use crate::error::{empty_err, EmptyArg, Error, Panic, Utf8Err, WasmErr};
-use cosmwasm_std::Extern;
+use cosmwasm_vm::Extern;
```

Comment:
Merge with line below? No linter complaining here?

Reply:
Oh, must've missed it. I did a round of cleaning up the imports before pushing the commit. I'll fix it in the next one